src/Encryption/Codec/EncryptionCodec.cs

```diff
@@ -14,26 +14,18 @@ public sealed class EncryptionCodec : IPayloadCodec, IDisposable
     private const int TagSize = 16;
     private static readonly ByteString EncodingByteString = ByteString.CopyFromUtf8("binary/encrypted");
 
+    private readonly byte[] key;
     private readonly ByteString keyIDByteString;
-    private readonly AesGcm aes;
 
-    public EncryptionCodec(string keyID = DefaultKeyID, byte[]? key = null)
+    public EncryptionCodec(string keyID = DefaultKeyID, byte[]? key)
```

Comment:
```suggestion
    public EncryptionCodec(string keyID = DefaultKeyID, byte[]? key = null)
```

This is still breaking compilation because the default value was removed.